modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -3646,8 +3646,6 @@ public boolean isDataTypeFile(String dataType) {
                     // As of January 2020, the "signature" scheme has not been registered with IANA yet.
                     // This scheme may have to be changed when it is officially registered with IANA.
                     cs.isHttpSignature = true;
-                } else {
-                    throw new RuntimeException("Unsupported security scheme: " + securityScheme.getScheme());
```

Comment:
Why should we remove this exception?

Reply:
This is to address  @jimschubert comments:

> This would be a breaking change, as we failed silently for any unsupported schemes previously.

Reply:
@jimschubert doesn't this clause mean that we failed loudly before, not silently?
How did we fail silently before?

Reply:
I think @jimschubert  meant codegen ignores the scheme, rather that "fail".
Ideally, developers would be able to add their own security scheme without having to fork OpenAPITools. There would be a way to register a new security scheme.

Reply:
Ah, I see that from our master branch at https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java. I misunderstood and thought that master had `throw new RuntimeException` when it was that only the earlier commit in this branch had it.
Feel free to mark this as resolved. Thank you for the explanation.